src/ostorlab/scanner/callbacks.py

```diff
@@ -153,31 +165,73 @@ def _update_state_reporter(
     return state_reporter
 
 
-def start_scan(
+def _connect_to_containers_registry(
+    configuration: scanner_conf.RegistryConfig,
+) -> docker.DockerClient:
+    """Connect to container registry."""
+    client = docker.from_env()
+    client.login(
+        username=configuration.username,
+        password=configuration.token,
+        registry=OSTORLAB_REGISTRY,
```

Comment:
:) this should be coming from the config, not hardcoded here.

Reply:
Worth doing in another iteration.

Reply:
It's a small change, already implemented.